### PR TITLE
Issue: Hotfix for Startup Script and Readme file

### PR DESCRIPTION
**Pull request by waji46**
**Title: Fixed issues in the  startup script, Updated the ReadMe file**
Link: #16

	[PS-1]
          > Fixed issues in startup Script Commands.
          > Updated Readme file content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,20 +95,23 @@ Check the below video for a quick demo of XC3.
...
-cd XC3/infrastructure/
-terraform init
-`
-    `bash
-Expected Output: It will create .terraform directory in XC3/infrastructure/  location
+    go to `xc3/infrastructure/` directory.
+    ```
+    cd xc3/infrastructure/
+    ``` 
+    And run below command
```
Comment (faheem-khau9) [inline README.md:110]:
```suggestion
    And run the following command
```

File: README.md [hunk 2]
```diff
@@ -166,24 +160,37 @@ Expected Output: It will create .terraform directory in XC3/infrastructure/  loc
 7.  Please copy msg_templates in custodian directory on deployed EC2 instance
 
     ```
-    scp -i "keypair.pem" keypair.pem bastion-host-dns:/directory-to-copy-keypair
+    sudo scp -i "keypair.pem" keypair.pem bastion-host-dns:/directory-to-copy-keypair
```
Comment (faheem-khau9) [inline README.md:163]: this keypair.pem is same as in the initial script?

File: infrastructure/modules/xc3/variables.tf [hunk 3]
```diff
@@ -87,7 +87,7 @@ variable "hosted_zone_id" {
 variable "cognito_domain" {
   type        = string
   description = "Cognito Domain"
-  default     = "grafana-domain"
+  default     = "wajahat-domain"
```
Comment (faheem-khau9) [inline infrastructure/modules/xc3/variables.tf:90]: Can you dynamically add the namespace with the domain? where cognito_domain is being used.

File: infrastructure/modules/serverless/cloudtrail.tf [hunk 4]
```diff
@@ -1,84 +1,93 @@
-# Copyright (c) 2023, Xgrid Inc, https://xgrid.co
+# # Copyright (c) 2023, Xgrid Inc, https://xgrid.co
```
[the remark this answers is not in the document]
Comment (samanXgrid) [reply infrastructure/modules/serverless/cloudtrail.tf:1]: no we don't need to remove it. It should be there

File: README.md [hunk 5]
```diff
@@ -166,24 +160,37 @@ Expected Output: It will create .terraform directory in XC3/infrastructure/  loc
...
     ```
+        source /custodian/bin/activate
+        cd ccloud_custodian_policies
+
      1. custodian run -s s3://${bucket_name}/iam-user --region ${aws_region} iam-user.yml
 
      2. custodian run -s s3://${bucket_name}/iam-role/ --region ${aws_region} iam-role.yml
 
-     3. custodian run -s tagging-compliance --region ${aws_region} eks-tagging.yml --region all
+     3. custodian run -s tagging-compliance --region ${aws_region} eks-tagging.yml
```
Comment (samanXgrid) [inline README.md:191]: why we are removing --region all . It is one time command and should be run once

File: README.md [hunk 6]
```diff
@@ -82,15 +82,6 @@ Check the below video for a quick demo of XC3.
    Note : if no private/public subnets provided then XC3 will create new VPC, private and public subnets and also XC3 will destroy these resources once
    user destroys XC3 setup.
 
-10. XC3 will create an EC2 instance during deployment, the user needs to create an AWS key_pair file in order to login to EC2 instance for troubleshooting purpose.
-11. If the ssh access is restricted only through bastion/jump server/vpn, user should have the security group ID of the bastion/jump/vpn EC2 instance.
-12. The user has to **enable CostExplorer** by following the below link.
-
-    https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/ce-enable.html
```
Comment (samanXgrid) [inline README.md:89]: why are we removing these lines

File: README.md [hunk 7]
```diff
@@ -82,15 +82,6 @@ Check the below video for a quick demo of XC3.
    Note : if no private/public subnets provided then XC3 will create new VPC, private and public subnets and also XC3 will destroy these resources once
    user destroys XC3 setup.
 
-10. XC3 will create an EC2 instance during deployment, the user needs to create an AWS key_pair file in order to login to EC2 instance for troubleshooting purpose.
-11. If the ssh access is restricted only through bastion/jump server/vpn, user should have the security group ID of the bastion/jump/vpn EC2 instance.
-12. The user has to **enable CostExplorer** by following the below link.
```
[the remark this answers is not in the document]
Comment (waji46) [reply README.md:87]: These points were repeated in ReadMe file.

File: README.md (hunk 6 above)
Comment (waji46) [reply README.md:89]: This line is repeated so, removed it.
image

File: README.md (hunk 2 above)
Comment (waji46) [reply README.md:163]: Yes, same as the initial script.
image

File: README.md (hunk 5 above)
Comment (waji46) [reply README.md:191]: When i run the command with this command, it was giving error, so i removed this flag and run the command, it worked fine for one region.

**Review by faheem-khau9 — APPROVED** [review]
LGTM

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by faheem-khau9 — CHANGES_REQUESTED** [review]
The Changelog file needs to be updated accordingly.

**Comment by waji46** [comment]
> The Changelog file needs to be updated accordingly.

@faheem-khau9 We can update the changeLog file for the new version, i have just made small changes in Readme and startup-script.

**Review by faheem-khau9 — APPROVED** [review]
LGTM

**Review by samanXgrid — APPROVED** [review]
LGTM

**Review by muhdsaad — APPROVED** [review]
LGTM

**Pull request merged**